Products/ZenModel/Organizer.py

```diff
@@ -183,6 +184,7 @@ def manage_addOrganizer(self, newPath, factory=None, REQUEST=None):
         if factory is None:
             factory = self.__class__
         if not newPath: return self.callZenScreen(REQUEST)
+        newPath = self.prepId(newPath)
```

Comment:
This can't be done here. It needs to be below this. if newPath is a path with slashes, those slashes will be sanitized.